### PR TITLE
test: add tests for view command

### LINKED ISSUE #103: test: create test files for ViewCommand

### PR DESCRIPTION
**Pull request by Oddcorner**
Closes #103

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 1]
```diff
@@ -81,8 +81,8 @@ public boolean equals(Object o) {
         }
 
         ViewCommand other = (ViewCommand) o;
-        return targetName.equals(other.targetName)
-                || targetIndex.equals(other.targetIndex); // state check
+        return (targetName != null && targetName.equals(other.targetName))
+                || (targetIndex != null && targetIndex.equals(other.targetIndex)); // state check
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:85]: Nice way to handle the equality of this. We'll have to keep this in mind for the other .equals methods that uses name/index flags.

**Review by yusufaine — APPROVED** [review]
LGTM! 🌮 🔥

File: src/main/java/seedu/address/logic/commands/ViewCommand.java (hunk 1 above)
Comment (blaketingyu) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:85]: okay will keep this in mind for the other .equals methods that uses name/index flags.

**Review by blaketingyu — APPROVED** [review]
LGTM BRO LETS GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO 🌮 🦖

**Review by aidanaden — APPROVED** [review]
LGTM! lessgedit 🍝 🍝 🍝

**Pull request merged**